edn_format/edn_dump.py

```diff
@@ -42,13 +42,19 @@ def replace(match):
         return ESCAPE_DCT[match.group(0)]
     return '"' + ESCAPE.sub(replace, string) + '"'
 
-def seq(obj, string_encoding = DEFAULT_INPUT_ENCODING,
-             keyword_keys = False):
-    return ' '.join([udump(i, string_encoding=string_encoding,
-                              keyword_keys=keyword_keys) for i in obj])
+def seq(obj, **kw):
+    return ' '.join([udump(i, **kw) for i in obj])
 
 def udump(obj, string_encoding = DEFAULT_INPUT_ENCODING,
-               keyword_keys = False):
+               keyword_keys = False,
+               sort_keys = False):
+
+    kw = dict(
```

Comment:
Just laziness; it’s easier to write e.g. `dict(a=a, b=b, c=c)` rather than `{"a": a, "b": b, "c": c}`. I changed to a dict literal.